src/app/components/InfoBox/index.tsx

```diff
@@ -22,11 +22,8 @@ export function InfoBox(props) {
   }
 
   return (
-    <div
-      className="container mb-3"
-      style={{ display: show ? 'block' : 'none' }}
-    >
-      <div className="p-3 bg-component-bg">
+    <div className="mt-3" style={{ display: show ? 'block' : 'none' }}>
```

Comment:
Direct style manipulation is discouraged for react,  so I would recommend on using classes instead:
```suggestion
    <div className={`mt-3 d-none ${show && 'd-nlock'}`}>
```